documentation/modules/proc-metrics-deploying-prometheus.adoc

```diff
@@ -29,7 +29,9 @@ On MacOS, use:
 [source,shell,subs="+quotes,attributes"]
 sed -i '' 's/namespace: .*/namespace: _my-namespace_/' prometheus.yaml
 
-. Edit the `ServiceMonitor` resource in `strimzi-service-monitor.yaml` to define Prometheus jobs that will scrape the metrics data.
```

Comment:
Do we want to say something in the introduction to this procedure about service and pod metrics data? Examples?